### PR TITLE
Passive hermits

### PR DESCRIPTION
**Pull request by Tyrannicodin**
Adds passive hermits:
- [x] Postmaster Pearl
- [x] Cyberpunk Impulse
- [x] Zookeeper Scar

Todo:
- [x] Tests
- [x] Update Postmaster Pearl to use shuffle deck function for cards, rather than discarding.
- [x] Update stats for all hermits

### CODE REVIEW

**Comment by Tyrannicodin** [comment]
Ready to merge after Screapear makes his changes to Zookeeper scar.

**Review by Screaper91 — CHANGES_REQUESTED** [review]
- My bugfixes for passives are waiting in [Tyrannicodin/hc-tcg#28](https://github.com/Tyrannicodin/hc-tcg/pull/28)
- All three hermits share the same numericId _!_
- **Bop 'N' Go** needs additional work to support Stratos Joel, DM Tango, and/or Hypno interacting with "attached" items

File: tests/unit/game/advent-of-tcg/hermits/postmasterpearl-rare.test.ts
[the remark this answers is not in the document]
Comment (alicetouhou) [reply tests/unit/game/advent-of-tcg/hermits/postmasterpearl-rare.test.ts:1]: If it's not deterministic right now that should probably be changed. We need it to be deterministic for replays to work

[the remark this answers is not in the document]
Comment (Tyrannicodin) [reply tests/unit/game/advent-of-tcg/hermits/postmasterpearl-rare.test.ts:1]: I thougt Lunar changed it to be deterministic.

[the remark this answers is not in the document]
Comment (Lunarmagpie) [reply tests/unit/game/advent-of-tcg/hermits/postmasterpearl-rare.test.ts:1]: It is deterministic! But to be honest relying on rng in a test kinda rubs me the wrong way. I would refactor it.

[the remark this answers is not in the document]
Comment (Tyrannicodin) [reply tests/unit/game/advent-of-tcg/hermits/postmasterpearl-rare.test.ts:1]: I'm not sure how to test it a different way - any suggestions?

[the remark this answers is not in the document]
Comment (Lunarmagpie) [reply tests/unit/game/advent-of-tcg/hermits/postmasterpearl-rare.test.ts:1]: To be honest we're getting close to the deadline so I think its okay to skip the test for this and we can deal with it later.

**Review by Screaper91 — COMMENTED** [review]
This can be merged once the postmaster pearl tests are disabled. Missing assets to add soon:
- "Passive!" logo
- Post Inspector status icon `client/public/images/status/post.png`

**Pull request closed without merging**